src/libsyntax/feature_gate.rs

```diff
@@ -479,6 +479,9 @@ declare_features! (
 
     // Allows async and await syntax
     (active, async_await, "1.28.0", Some(50547), None),
+
+    // Allows async and await syntax
+    (active, cfg_target_has_atomic_cas, "1.28.0", Some(0), None),
```

Comment:
after this gets a thumbs up I'll create a tracking issue and fill it the number here